src/main/java/com/iexec/standalone/configuration/PublicConfigurationService.java

```diff
@@ -87,4 +97,28 @@ public PublicConfiguration getPublicConfiguration() {
         }
         return publicConfiguration;
     }
+
+    public String getSchedulerPublicAddress() {
+        return publicConfiguration.getSchedulerPublicAddress();
+    }
+
+    public String getRequiredWorkerVersion() {
+        return publicConfiguration.getRequiredWorkerVersion();
+    }
+
+    @Bean
+    public ConfigServerClient configServerClient() {
+        final String configServerURL = StringUtils.isBlank(publicConfiguration.getConfigServerUrl()) ?
+                publicConfiguration.getBlockchainAdapterUrl() : publicConfiguration.getConfigServerUrl();
+        return ConfigServerClientBuilder.getInstance(
+                Logger.Level.NONE,
+                configServerURL);
+    }
```

Comment:
A `ConfigServerClient` bean is already created in `ConfigServerClientConfig` class.
This code will lead to an error at startup.